docs/api/proposed_endpoints.md

```diff
@@ -0,0 +1,186 @@
+[TOC]
+
+# Posts related actions
+
+## Create a post
+
+Create a post of contents.
+
+```
+POST api/posts
+```
+
+**Paramaters**
+
+Name         | Description
+-------------|--------------------------------------
+id           | identity of the post
+posted_by    | user who is posting the content
+posted_at    | social media ploatform to post
+posted_time  | time at which post should be posted
+content_type | type of content, text or image
+content      | content of the post
+image        | image in the post
+is_approved  | whether the post is approved or not
+is_posted    | whether the post is posted or not
+
+**Request**
+```json
+{
+    "id": "724",
+    "posted_by": "John Micky",
+    "posted_at": "Twitter",
+    "posted_time": "",
+    "content_type": "text",
+    "content": "Anything that user has to write",
+    "image": "",
+    "is_approved": "False",
+    "is_posted": "False"
+}
+```
+
+**Response**
+```json
+
+Status: 201 Created
+{
+    "id": "724",
+    "posted_by": "John Micky",
+    "posted_at": "Twitter",
+    "posted_time": "",
+    "content_type": "text",
+    "content": "Anything that user has to write",
+    "image": "http://xyz.com/path/of/image.jpg",
+    "is_approved": "False",
```

Comment:
Why is this string `False`?

This should be `false` in JS. This is JSON format.